cinema_service/settings.py

```diff
@@ -140,3 +141,27 @@
 # https://docs.djangoproject.com/en/4.0/ref/settings/#default-auto-field
 
 DEFAULT_AUTO_FIELD = "django.db.models.BigAutoField"
+
+REST_FRAMEWORK = {
+    "DEFAULT_AUTHENTICATION_CLASSES": (
+        "rest_framework_simplejwt.authentication.JWTAuthentication",
+    ),
+    "DEFAULT_PERMISSION_CLASSES": (
+      "cinema.permissions.IsAdminOrIfAuthenticatedReadOnly",
+    ),
+
+    "DEFAULT_SCHEMA_CLASS": "drf_spectacular.openapi.AutoSchema",
+    "DEFAULT_THROTTLE_CLASSES": [
+        "rest_framework.throttling.AnonRateThrottle",
+        "rest_framework.throttling.UserRateThrottle"
+    ],
+    "DEFAULT_THROTTLE_RATES": {
+        "anon": "10/min",
+        "user": "30/min"
+    }
+
+}
+
+DEFAULT_AUTHENTICATION_CLASSES = (
+    "rest_framework_simplejwt.authentication.JWTAuthentication",
+)
```

Comment:
The `DEFAULT_AUTHENTICATION_CLASSES` is defined twice, once inside `REST_FRAMEWORK` and once outside. Consider removing the redundant definition to avoid confusion.